v20.1/cockroach-start.md

```diff
@@ -485,6 +485,14 @@ $ cockroach sql --insecure --port 26261
 (1 row)
 ~~~
 
+### Start a cluster with separate RPC and SQL networks
+
+Start a cluster with separate RPC and SQL networks to avoid bandwidth interference and add a level of isolation against network attacks as a measure of 'defense in depth'.
```

Comment:
Protection against bandwidth interference is really only relevant when using different addresses.

Reply:
How about "Start a cluster with separate RPC and SQL networks to partition traffic and add a level of isolation against network attacks as a measure of 'defense in depth'."